validation/l2oo_test.go

```diff
@@ -0,0 +1,159 @@
+package validation
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"testing"
+
+	"github.com/ethereum-optimism/optimism/op-bindings/bindings"
+	. "github.com/ethereum-optimism/superchain-registry/superchain"
+	"github.com/stretchr/testify/require"
+
+	"github.com/ethereum-optimism/optimism/op-service/retry"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethclient"
+)
+
+func TestL2OOParams(t *testing.T) {
+
+	isExcluded := map[uint64]bool{
+		10:           true, // mainnet/op                      (old version of L2OutputOracle, no submissionInterval method)
```

Comment:
Notice most chains have to be excluded for this reason. Note also that[ the old contract does expose a `SUBMISSION_INTERVAL` getter. ](https://etherscan.io/address/0xdfe97868233d1aa22e815a266982f2cf17685a27#readProxyContract). I can add a workaround for this, but I wasn't sure if it was worth it (presumably future chains will have the newer version of the contract)?

Reply:
These chains will be upgraded in the future to include that getter, and once FPs launch the L2OO will be replaced altogether. I'm fine with this solution for now.